### PR TITLE
Mnemonic and seed implement zeroize.

### LINKED ISSUE #19: Add a mechanism to prevent the leak of sensitive data
[Seed](https://github.com/maciejhirsz/tiny-bip39/blob/master/src/seed.rs#L21) contains sensitive data that can remain in memory even after it is dropped. for that reason, such data should be zeroized.

### PR DESCRIPTION
**Pull request by realbigsean**
Resolves #19 

Extends #20 to add zeroization for `Mnemonic`

Changes:
- `Seed` implements zeroize
- `Mnemonic` implements zeroize
- `into_phrase` updated to accommodate zeroize on drop

### CODE REVIEW

**Comment by paulhauner** [comment]
I'd love to see the merged into master :)

File: src/mnemonic.rs [hunk 1]
```diff
@@ -220,10 +225,12 @@ impl Mnemonic {
...
     /// Consume the `Mnemonic` and return the phrase as a `String`.
-    ///
-    /// This operation doesn't perform any allocations.
-    pub fn into_phrase(self) -> String {
-        self.phrase
+    pub fn into_phrase(mut self) -> String {
+        // Create an empty string and swap values with the mnemonic's phrase.
+        // This allows `Mnemonic` to implement `Drop`, while still returning the phrase.
+        let mut phrase = String::new();
+        mem::swap(&mut self.phrase, &mut phrase);
```
Comment (maciejhirsz) [inline src/mnemonic.rs:232]:
```suggestion
        mem::swap(&mut self.phrase, String:new());
```

**Review by maciejhirsz — APPROVED** [review]

File: src/mnemonic.rs (hunk 1 above)
Comment (maciejhirsz) [reply src/mnemonic.rs]: I borked this suggestion, just woke up. [Fixed on master](https://github.com/maciejhirsz/tiny-bip39/commit/7d807b02fb6710ad42e0934956665f7ba0d3b80d).

**Comment by maciejhirsz** [comment]
Cheers, this is now released in 0.8.0.

**Pull request merged**